cmd/storagenode-updater/main.go

```diff
@@ -96,119 +108,110 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 		cancel()
 	}()
 
-	loopInterval, err := time.ParseDuration(interval)
-	if err != nil {
-		return errs.New("unable to parse interval parameter: %v", err)
-	}
-
 	loopFunc := func(ctx context.Context) (err error) {
-		if err := update(ctx); err != nil {
+		if err := update(ctx, ident.ID); err != nil {
 			// don't finish loop in case of error just wait for another execution
 			log.Println(err)
```

Comment:
this most probably will print `nil` to logs.

Reply:
How so, this line should only be reached if `err != nil`?

Reply:
yes, `update` can return `nil` do Println will display it.

Reply:
ignore my comment, I'm talking nonsense :)